playbooks/freeipa/tasks/main.yaml

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
Would it be more idiomatic to add this as a role and include it in `playbooks/freeipa.yml`?